spring-cloud-gateway-core/src/test/resources/application.yml

```diff
@@ -253,6 +253,15 @@ spring:
         filters:
         - RequestHeaderToRequestUri=X-CF-Forwarded-Url
 
+      # =====================================
+      - id: cloudfoundry_routeservice_test
+        uri: ${test.uri}
+        predicates:
+        - CloudFoundryRouteService=
+        - Header=X-CF-Forwarded-Url
```

Comment:
`Header` predicate shouldn't be necessary, it's already covered in the `CloudFoundryRouteService` predicate

Reply:
Removed. Thanks